lib/exercises.rb

```diff
@@ -1,19 +1,64 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n^2)
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 Interesting solution with `char.ord`   This works if you know that all the letters are a-z and roughly the same length.  That's because you could end up with two strings of different lengths that have the same sum of characters.